### PR TITLE
Update distribution and module versions

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose

Related to: https://github.com/ballerina-platform/ballerina-standard-library/issues/2478

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by shafreenAnfar — APPROVED** [review]

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [package]
 org = "ballerinax"
 name = "nats"
-version = "2.0.2"
+version = "2.1.0"
```
Comment (niveathika) [inline ballerina/Ballerina.toml:4]: This needs to be changed in gradle.properties

**Review by niveathika — CHANGES_REQUESTED** [review]

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (niveathika) [reply ballerina/Ballerina.toml:4]: Lets also run the build and update the toml files

Comment (aashikam) [reply ballerina/Ballerina.toml]: Done.

Comment (aashikam) [reply ballerina/Ballerina.toml]: Could you check if it is the correct toml file, that has been updated

File: build-config/resources/Ballerina.toml [hunk 2]
```diff
@@ -6,7 +6,7 @@ authors = ["Ballerina"]
 keywords = ["service", "client", "messaging", "network", "pubsub"]
 repository = "https://github.com/ballerina-platform/module-ballerinax-nats"
 license = ["Apache-2.0"]
-distribution = "slbeta3"
+distribution = "slbet6"
```
Comment (Maninda) [inline build-config/resources/Ballerina.toml:9]:
```suggestion
distribution = "slbeta6"
```

**Review by niveathika — APPROVED** [review]

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**